src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/WPF/AppDotXamlDocument.cs

```diff
@@ -0,0 +1,944 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Xml;
+using Microsoft.VisualStudio.TextManager.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.WPF;
+
+/// <summary>
+/// When reading and writing the Application.xaml file, we have a requirement that
```

Comment:
```suggestion
/// When reading and writing the App.xaml file, we have a requirement that
```

Reply:
The file name is actually Application.xaml, not App.xaml.

Reply:
Is it not this file?

![image](https://user-images.githubusercontent.com/350947/171063998-a2d26ec8-687b-4074-bf11-536fe828805e.png)